ledger/participant-integration-api/src/main/scala/platform/apiserver/execution/CommandExecutionResult.scala

```diff
@@ -25,4 +26,5 @@ private[apiserver] final case class CommandExecutionResult(
     transaction: SubmittedTransaction,
     dependsOnLedgerTime: Boolean,
     interpretationTimeNanos: Long,
+    globalKeyMapping: Map[GlobalKey, Option[Value.ContractId]],
```

Comment:
What would be the difference of having None vs no value in the map in this case? 
As I assume a good reason, I would appreciate a comment explaining it

Reply:
The map should contain all contract keys that were used during interpretation. A value of `None` means no contract was found with this contract key.

I agree. that it would be good to a comment for this new field.

Reply:
maybe an ADT instead of an Option would be a better choice here?

Reply:
Added comment. How about not going for an ADT to keep this aligned with Engine: it is very much a pass-through from Engine to Ledger-Backend (In-between mapping might raise confusion)?

Reply:
I'm also in favour of leaving this in line with what's used in the Engine.